e2e/specs/formsDashboard.spec.ts

```diff
@@ -0,0 +1,101 @@
+import { test } from "../core";
+import { expect } from "@playwright/test";
+import { FormBuilderPage } from "../pages";
+import { deleteForm } from "../commands/formOperations";
+
+let formUuid = null;
+
+test("should filter forms based on publish status", async ({ page }) => {
+  const formBuilderPage = new FormBuilderPage(page);
+  await formBuilderPage.gotoFormBuilder();
+
+  await formBuilderPage.createNewFormButton().click();
+
+  // Inputs the dummy schema
+  await formBuilderPage.inputDummySchemaButton().click();
+
+  // Save the form
+  await formBuilderPage.saveForm();
+  await formBuilderPage.publishFormButton().click();
+  await formBuilderPage.gotoFormBuilder();
```

Comment:
@hadijahkyampeire shall we use the API to create a form instead of the UI because this is being test by create form test we don't need to test it again. 

Refer to this,

https://github.com/openmrs/openmrs-esm-form-builder/blob/main/e2e/specs/editForm.spec.ts

Reply:
@anjula-sack you are right I can leverage that created form for my tests.